18F/identity-idp
# LG-11427: add phone_with_camera to analytics

## 🎫 Ticket
[LG-11427](https://cm-jira.usa.gov/browse/LG-11427)

## 🛠 Summary of changes
Store phone_with_camera in idv_session and log in analytics for IdV events.

## 📜 Testing Plan
Bypass phone question page
1. Complete identity verification
2. verify phone_with_camera=nil is logged for most Idv events post agreement step.

Phone question page with camera
1. Complete Idv steps until hybrid handoff page is rendered
3. visit /verify/phone_question
4. Select I have a phone and complete Idv
5. verify phone_with_camera=true is logged for most Idv events post agreement step.

Phone question page without camera
1. Complete Idv steps until hybrid handoff page is rendered
3. visit /verify/phone_question
4. Select I do have a phone and complete Idv
5. verify phone_with_camera=false is logged for most Idv events post agreement step.

NOTE: events not logged
hybrid document capture events
image upload submitted
image upload pii validation
